wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/commandbasedbestpracticeled/PeriodicTask.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot;
+
+/**
+Wrapper for TimeRobot.addPeriodic() to pass for other classes usage
+
+Example Usage:
+file Robot.java:
+new RobotContainer(this::addPeriodic);
+
+file RobotConatiner.java:
+RobotContainer(PeriodicTask periodicTask) {
+   periodicTask.register( ()->System.out.println("periodically printing"), 1., 0.01);
+  }
+
+ */
+@FunctionalInterface
+public interface PeriodicTask {

```

Comment:
I feel like this should be part of WPILIB API surface if this pattern is an officially blessed mechanism of injecting periodic processing support.

Reply:
> if this pattern is an officially blessed mechanism of injecting periodic processing support

It's not. We have TimedRobot.addPeriodic() instead.